### PR TITLE
Add support for nocaptcha recaptcha (recaptcha2)

### PR DESCRIPTION
**Pull request by cdvv7788**
Add NOCAPTCHA=True to your settings. That should be enough to switch to the new api. Defaults to ssl, so the ssl related setting is ignored.

### CODE REVIEW

**Comment by Hedde** [comment]
+1 for merging this asap

**Comment by uKev** [comment]
I've tested it in a project and it works really fine.
+1 for merging from here

File: captcha/templates/captcha/widget_nocaptcha.html [hunk 1]
```diff
@@ -0,0 +1,34 @@
...
+        RecaptchaOptions = DjangoRecaptchaOptions;
+    } else {
+        for (key in DjangoRecaptchaOptions) {
+            RecaptchaOptions[key] = DjangoRecaptchaOptions[key];
+        }
+    }
+</script>
+<div class="g-recaptcha" data-sitekey="{{ public_key }}"></div>
+<noscript>
+  <div style="width: 302px; height: 352px;">
```
Comment (Hedde) [inline captcha/templates/captcha/widget_nocaptcha.html]: +1

File: captcha/templates/captcha/widget_nocaptcha.html [hunk 2]
```diff
@@ -0,0 +1,33 @@
+<script src='https://www.google.com/recaptcha/api.js'></script>
```
Comment (dotsbb) [inline captcha/templates/captcha/widget_nocaptcha.html]: Language setting should be set as `hl` GET param according the new [docs ](http://developers.google.com/recaptcha/docs/display#config)

**Comment by intellisense** [comment]
This does not work if `CAPTCHA_AJAX=True` it renders old widget template. Anyone had this issue? here is my config:

```
RECAPTCHA_PUBLIC_KEY = os.environ.get('RECAPTCHA_PUBLIC_KEY', '')
RECAPTCHA_PRIVATE_KEY = os.environ.get('RECAPTCHA_PRIVATE_KEY', '')
NOCAPTCHA = True
RECAPTCHA_USE_SSL = os.environ.get('SITE_USES_SSL', False)  # I know this is ignored if using NOCAPTCHA
CAPTCHA_AJAX = True
```

**Comment by abarax** [comment]
@intellisense I had this issue also. Did you find a work around?

**Pull request merged**